shell/platform/fuchsia/flutter/pointer_injector_delegate.cc

```diff
@@ -129,7 +129,7 @@ bool PointerInjectorDelegate::HandlePlatformMessage(
     }
 
     auto timestamp = args.FindMember("timestamp");
-    if (!timestamp->value.IsInt()) {
+    if (!timestamp->value.IsInt() && !timestamp->value.IsUint64()) {
```

Comment:
Any unit test we can add for this?

Reply:
We already have some unittests in pointerinjector_delegate_unittest.cc which check this code path.

Reply:
Actually those were added as part of the previous PR (https://github.com/flutter/engine/pull/34692). This was a minor bug which I discovered while doing some manual testing on estelle.

Reply:
I see, thanks, sorry for not checking first.